### PR TITLE
Create permissions for EVMScriptRegistry roles

### LINKED ISSUE #23: Create permissions for EVMScriptRegistry roles in all kits
https://github.com/aragon/aragonOS/pull/392 splits the roles of `EVMScriptRegistry` into two: `REGISTRY_ADD_EXECUTOR_ROLE` and `REGISTRY_MANAGER_ROLE`.

Now `EVMScriptRegistryFactory` completely removes permissions after setting it up. These permissions are too dangerous to leave uninitialized as anyone with the `CREATE_PERMISSIONS_ROLE` could create them at any time.

Kits should create the permissions for both roles and assign them to an entity with significant power (for beta kits, that would be the Voting app)

### PR DESCRIPTION
**Pull request by bingen**
Closes #23.

### CODE REVIEW

File: kits/bare/contracts/KitBase.sol [hunk 1]
```diff
@@ -4,9 +4,10 @@ import "@aragon/os/contracts/factory/DAOFactory.sol";
 import "@aragon/os/contracts/apm/Repo.sol";
 import "@aragon/os/contracts/lib/ens/ENS.sol";
 import "@aragon/os/contracts/lib/ens/PublicResolver.sol";
+import "@aragon/os/contracts/evmscript/IEVMScriptRegistry.sol"; // needed for EVMSCRIPT_REGISTRY_APP_ID
 
 
-contract KitBase {
+contract KitBase is EVMScriptRegistryConstants {
```
[the remark this answers is not in the document]
Comment (bingen) [reply kits/bare/contracts/KitBase.sol]: Well, I put it here to use it in `Survey` too. Your comment made me realize that Survey was not inheriting from `KitBase` yet, thanks! :-) (Fixing it)

File: kits/beta-base/test/docker.js [hunk 2]
```diff
@@ -49,6 +50,41 @@ contract('Beta Base Kit', accounts => {
         assert.notEqual(votingAddress, '0x0', 'Voting not generated')
     })
 
+    it('has correct permissions', async () =>{
+        const dao = await getContract('Kernel').at(daoAddress)
+        const acl = await getContract('ACL').at(await dao.acl())
+
+        // app manager role
+        assert.equal(await acl.getPermissionManager(daoAddress, (await dao.APP_MANAGER_ROLE())), votingAddress, 'App manager role manager should match')
+
```
[the remark this answers is not in the document]
Comment (bingen) [reply kits/beta-base/test/docker.js]: I'll do that in Democracy and Multisig kits. Here `CREATE_PERMISSION_ROLE` is kind of dangling.

[the remark this answers is not in the document]
Comment (sohkai) [reply kits/beta-base/test/docker.js]: Right, reminded me that we should probably bit a little bit of documentation / comments to explain that the inherited kits from `BetaKitBase` need to clean up that permission!

File: kits/democracy/test/democracy.js [hunk 3]
```diff
@@ -121,6 +121,12 @@ contract('Democracy Kit', accounts => {
             // create permissions role
             await checkRole(acl.address, await acl.CREATE_PERMISSIONS_ROLE(), votingAddress, 'ACL', 'CREATE_PERMISSION')
 
+            // evm script registry
+            const regConstants = await getContract('EVMScriptRegistryConstants').new()
+            const reg = await getContract('EVMScriptRegistry').at(await dao.getApp(await dao.APP_ADDR_NAMESPACE(), (await regConstants.EVMSCRIPT_REGISTRY_APP_ID())))
+            await checkRole(reg.address, await reg.REGISTRY_ADD_EXECUTOR_ROLE(), votingAddress, 'EVMScriptRegistry', 'ADD_EXECUTOR')
+            await checkRole(reg.address, await reg.REGISTRY_MANAGER_ROLE(), votingAddress, 'EVMScriptRegistry', 'REGISTRIY_MANAGER')
```
Comment (izqui) [inline kits/democracy/test/democracy.js]: Typo in the role name: `REGISTRIY_MANAGER`.

**Pull request merged**